proto/asgt/data/v1alpha1/create.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package asgt.dataservice.v1alpha1;
+
+option go_package = "data";
+
+import "asgt/type/sample.proto";
+import "asgt/type/retention_policy.proto";
+
+message CreateRequest {
+  string name = 1;
+  string type = 2;
+  repeated string tags = 3;
+  repeated string targets = 6;
+  repeated asgt.type.Sample samples = 4;
+  asgt.type.RetentionPolicy retention_policy = 5;
+}
+
+message AppendDataRequest {
```

Comment:
Does this belong in create.proto ? 
Do we need a create.proto and delete.proto.
the request messages could just live in data_service.proto couldn't they?

Reply:
Or otherwise it seems append should have a separate x.proto as well.

Reply:
My laziness. They no longer have to be in separate files, so moving them to data_service.proto.